### PR TITLE
Update loose binary idenfication docs for current UX

### PR DESCRIPTION
**Pull request by twsouthwick**

### CODE REVIEW

File: docs/design/Loose_binary_identification.md [hunk 1]
```diff
@@ -18,21 +18,33 @@ The main scenario this will focus on is one where a project has downloaded an as
...
+
+1. Set an environment variable for `UA_FEATURES=EXTENSION_MANAGEMENT`. Extension management is currently under preview, but is required to test this feature.
+2. Restore the extensions installed
+   ```
+   upgrade-assistant extensions restore
+   ```
+
+In order to update to the latest versions, the extensions may be updated:
+
+3. Set an environment variable for `UA_FEATURES=EXTENSION_MANAGEMENT`. Extension management is currently under preview, but is required to test this feature.
```
Comment (sunandabalu) [inline docs/design/Loose_binary_identification.md:43]: Do the customers need to set this environment variable 3 times? looks that way from the documentation here. Could you make it clear if the setting will persist?

**Review by sunandabalu — CHANGES_REQUESTED** [review]
This section needs to be a separate usage scenario doc including command to run to analyze and upgrade using the extensions, along with the pre-setup for the commands to work (set env variable, download extensions etc.)

File: docs/design/Loose_binary_identification.md (hunk 1 above)
Comment (twsouthwick) [reply docs/design/Loose_binary_identification.md]: That's up to the user. It's an environment variable and they can manage that however they want

**Comment by twsouthwick** [comment]
So you'd like a separate doc for usage of the extension without the context of what it's supposed to do?

**Comment by sunandabalu** [comment]
> So you'd like a separate doc for usage of the extension without the context of what it's supposed to do?

you can always have "For more context on how extensions work see here..." at the beginning of the doc. That way the usage doc is specific to the scenario - "how to resolve loose assemblies in the project" , using extensions is an implementation detail that customers may not be interested in.

**Comment by twsouthwick** [comment]
The doc right now focuses on the the experience of identifying loose dependencies. Can you provide an outline of what you would like to see?

**Comment by sunandabalu** [comment]
With the `How to use` section in the Loose-Assembly doc, I think its mostly there. I'm thinking about a line under https://github.com/dotnet/upgrade-assistant#analyze-solution-prior-to-upgrade to say "Loose-Assembly resolution to a corresponding package (if extensions are added as described here (link to "how to use" section)" Also a similar line talking about upgrade under https://github.com/dotnet/upgrade-assistant#upgrade-solution. This way we are only sharing one top level section to customers.

**Comment by twsouthwick** [comment]
So the section in `Experimental Features` you want duplicated in each of those sections? How do you propose to clearly mark that it's experimental and not GA'd?

**Review by sunandabalu — APPROVED** [review]
Fine for now.

**Pull request merged**